cli/libreant_db.py

```diff
@@ -95,5 +96,93 @@ def export_all(pretty):
     click.echo(json.dumps(volumes, indent=indent))
 
 
+@libreant_db.command(name='append-file', help='uploads a file to an existing volume')
+@click.option('-i', '--volumeid', type=click.STRING, )
+@click.option('-f', '--filepath', type=click.Path(exists=True,resolve_path=True), multiple=True, help='the path to the media to be uploaded')
+@click.option('-n', '--name', type=click.STRING, multiple=True, help='[optional] name of the media including the extension')
+@click.option('-m', '--mime', type=click.STRING, multiple=True, help='[optional] mime type of the media')
+@click.option('-t', '--notes', type=click.STRING, multiple=True, help='[optional] notes about the media')
+def append_file(volumeid,filepath,name,mime,notes):
```

Comment:
I like `append_file` more than `insert_volume`! Anyway, shouldn't `volumeid` and `filepath` be arguments instead of options? The thing about `[optional]` apply here, too.

Another thing is about the naming: volumes contains attachments, not files.